cid.go

```diff
@@ -79,6 +79,28 @@ const (
 	ZcashTx            = 0xc1
 )
 
+// Codecs maps the name of a codec to its type
+var Codecs = map[string]byte{
```

Comment:
I am not sure that byte is right format for it.
The codecs are not limited to a byte (due to varint) we just are trying to keep codecs that are being used in cid in the 8 bit range but it might not be much longer till we will have two byte CID codecs (we already have two and three byte multihashes).